tests/functional/storageclass/test_rbd_default_storageclass.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from ocs_ci.helpers.helpers import is_rbd_default_storage_class
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.helpers.helpers import default_storage_class
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    green_squad,
+    polarion_id,
+    baremetal_deployment_required,
+    ui_deployment_required,
+)
+
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
+@baremetal_deployment_required
```

Comment:
We need to use @vsphere_platform_required and @baremetal_deployment_required but it wouldn't work for both so I'm suggesting creating a new mark `barmetal_or_vsphere_required` in marks.py

Reply:
Maybe this will help `@on_prem_platform_required`.
It lead to this list in the end. The question is if the feature is supported for all those platform. We can check with DEV. If not let's create are own marker for vsphere and baremetal as described above.
`ON_PREM_PLATFORMS = [
    VSPHERE_PLATFORM,
    BAREMETAL_PLATFORM,
    BAREMETALPSI_PLATFORM,
    IBM_POWER_PLATFORM,
    RHV_PLATFORM,
    HCI_BAREMETAL,
    HCI_VSPHERE,
]
`

Reply:
I have created a marker `baremetal_or_vsphere_deployment_required ` in to the tests.